File: Selenium/Activities/activity12.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class activity12 {
    public static void main(String args[]) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/selenium/DynamicControls");
        System.out.println("Home page title: " + driver.getTitle());
        WebElement inputTextBox = driver.findElement(By.xpath("//input[@id='textInput']"));
        System.out.println("Input text box is Enabled? " + inputTextBox.isEnabled());
        driver.findElement(By.xpath("//button[contains(text(),'Enable Input')]")).click();
        System.out.println("Input text box is Enabled? " + inputTextBox.isEnabled());
        driver.quit();
    }
}

File: Selenium/Activities/activity12.py
from selenium.webdriver.common.by import By
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))
driver.get("https://training-support.net/selenium/DynamicControls")
print("Home page title: " + driver.title);
inputTextBox = driver.find_element(By.XPATH,"//input[@id='textInput']")
print(inputTextBox.is_enabled())
driver.find_element(By.XPATH,"//button[contains(text(),'Enable Input')]").click()
print(inputTextBox.is_enabled())
driver.quit()
